java/spacewalk-java.spec

```diff
@@ -189,6 +189,8 @@ BuildRequires:  tomcat6-lib
 %endif # 0{?suse_version}
 %endif # 0{?fedora} || 0{?rhel} >= 7
 Requires(pre):  salt
+#!BuildIgnore:  udev-mini libudev-mini1
+Requires(pre):  susemanager
```

Comment:
Shouldn't this be a Requires?

Reply:
This needs to be a `Requires(pre)` in this case. The `susemanager` group is actually created during the "pre" step of the `susemanager` package:

https://github.com/uyuni-project/uyuni/blob/master/susemanager/susemanager.spec#L197